sriov/common/configtestdata.py

```diff
@@ -41,15 +41,15 @@ def __init__(self, settings: Config) -> None:
         cpus = settings.config["dut"]["pmd_cpus"]
         self.container_cmd = (
             f"{settings.config['container_manager']} run -it --rm --privileged "
-            "-v /sys:/sys -v /dev:/dev -v /lib/modules:/lib/modules "
-            f"--cpuset-cpus {cpus} {dpdk_img} dpdk-testpmd -l {cpus} "
+            f"{settings.config['container_volumes']}"
+            f" --cpuset-cpus {cpus} {dpdk_img} dpdk-testpmd -l {cpus} "
```

Comment:
To make this string consistent with the rest of the format string, put the space at the end of the last line, not the beginning of this line.

Reply:
Addressed in most recent commit